plone-4.3.x.cfg

```diff
@@ -1,2 +1,2 @@
 [buildout]
-extends = http://dist.plone.org/release/4.3.9/versions.cfg
+extends = http://dist.plone.org/release/4.3.11-pending/versions.cfg
```

Comment:
Should we stick to released versions? In some days this link will no longer work, just like extends = http://dist.plone.org/release/4.3.10-pending/versions.cfg does not work... OTOH if there are some bits needed, then let's keep it but add an issue to update the link as soon as Eric does the final release